src/csp_post_processor/processor.py

```diff
@@ -1,12 +1,13 @@
 import hashlib
 import logging
 from collections import defaultdict
-from typing import Union
+from typing import List, Optional, Union
 
 from django.http import HttpRequest
 
 import html5lib
 import lxml.html
+import tinycss2
```

Comment:
we should make this dependency explicit - it's installed via WeasyPrint, but now it's a first part dependency.